harvest_cli/ocua.py

```diff
@@ -92,6 +95,12 @@ def useractions(start, end, wait, engage_host, user, password, output, queue_nam
     fail_count = 0
     last_action = None
 
+    try:
+        geolookup = Geolocate(geolite)
+    except IOError as e:
+        geolookup = None
+        logger.error("Error when reading maxminddb file: %s" % str(e))
```

Comment:
I think we want the process to do more than just log an error here. If it proceeds we get a bunch of records that we'd have to retroactively do geo-lookup on (which is possible), but I think it's better if it just fails to run.